# Parameter file for the supertoroid creation and for initial conditions for testing
# Intrinsic parameters a1, a2, a3, a4 control the size of the supertoroid, a1, a2 in x-y plane,
#a4 size of the hole in x-y plane, a3 in z direction.
# Intrinsic parameters e1, e2 are the exponents that control the shape, e2 in the x-y plane, 
#e1 in the z cross-section.


#Parameters for sampling the supertoroid
par_a1: 5.1
par_a2: 4.6
par_a3: 4.6
par_a4: 2.8
par_e1: 0.6
par_e2: 1.3 
par_x: 0.0
par_y: 0.0
par_z: 0.0
par_orx: 0.0
par_ory: 0.0
par_orz: 0.0

#Parameters for initial conditions of the solver
par_a10: 5
par_a20: 4
par_a30: 5
par_a40: 3
par_e10: 0.5
par_e20: 1
par_x0: 0.0
par_y0: 0.0
par_z0: 0.0
par_orx0: 0.0
par_ory0: 0.0
par_orz0: 0.0